### PR TITLE
Create matrixLab.yml

### PR DESCRIPTION
**Pull request by bvks2020sameer**
matrixLab is a set of C++ libraries to solve fundamental matrix functions matrixLab primarily constitutes of 5 different libraries

1. vector.h to handle all vector operations
2. matrix.h to handle all matrix operations
3. crammer.h to solve a set of linear equations
4. orthogonalization.h to orthogonalize a matrix
5. eigen.h to find eigen values of a matrix and upper triangular matrix

### CODE REVIEW

**Comment by bvks2020sameer** [comment]
matrixLab is a C++ implementation to solve various fundamental matrix problems or requirements that one may face during the development of various algorithms like machine learning, aerial robotics, etc. C++ being extremely fast over other languages like python , java, etc enables this library to be used in various real time applications. Also the cmake build enables cross platform development and development on arm based 64 bit embedded computers

**Comment by bvks2020sameer** [comment]
Added Documentation and python wrapper code

File: _data/projects/matrixLab.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+- matrix
+- mathematical-engine
+- cmake
+- msvc-2022
+- cross-platform
+- calculation
+- machine-learning
+upforgrabs:
+  name: enhancement
+  link: https://github.com/bvks2020sameer/matrixLab/issues?q=is%3Aissue+is%3Aopen+label%3Aenhancement
```
Comment (shiftkey) [inline _data/projects/matrixLab.yml]: :art: a more succinct label

```suggestion
  link: https://github.com/bvks2020sameer/matrixLab/labels/enhancement
```

**Comment by bvks2020sameer** [comment]
Thankyou for the merge . Really looking forward to take the project to a new Great Heights

**Pull request merged**